Classes/Backend/IndexingConfigurationSelectorField.php

```diff
@@ -102,12 +102,15 @@ public function getSelectedValues() {
 	/**
 	 * Renders a field to select which indexing configurations to initialize.
 	 *
-	 * Uses TCEforms.
+	 * Uses \TYPO3\CMS\Backend\Form\FormEngine.
 	 *
 	 *  @return string Markup for the select field
 	 */
 	public function render() {
-		$tceForm = t3lib_div::makeInstance('t3lib_TCEforms');
+		$selectFieldRenderer = $formEngine = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance('TYPO3\\CMS\\Backend\\Form\\FormEngine');
+		if (class_exists('TYPO3\\CMS\Backend\\Form\\Element\\SelectElement')) {
+			$selectFieldRenderer = \TYPO3\CMS\Core\Utility\GeneralUtility::makeInstance('TYPO3\\CMS\Backend\\Form\\Element\\SelectElement', $formEngine);
```

Comment:
SelectElement is new for 7.x
With this check it works also for 6.2 because there the SelectElement does not exist and it would use FormEngine to render as before.
I could also add a version check but I personally prefer feature checks. Maybe we could also check if formengine doesn't have the required render method?